tools/cli/admin.go

```diff
@@ -1280,5 +1280,14 @@ func newAdminConfigStoreCommands() []cli.Command {
 				AdminListDynamicConfig(c)
 			},
 		},
+		{
+			Name:    "describe",
```

Comment:
`list` seems to be a better name

Reply:
Yeah, naming is hard.

There is `list-dynamic-config` argument already, and I'm proposing to rename it to `list` [in this PR](https://github.com/uber/cadence/pull/5089/files#diff-44b0fcadee023fdd151d83020b84d464e980b288f49d165b1115f2dd43c075eaR1280)

Reply:
how about `listall`? `describe` makes me think that it is a command requiring a key as input and only shows the description of that config.

Reply:
done :+1: